WebRoot/messages/ZaMsg_ja.properties

```
@@ -1157,6 +1157,7 @@ LBL_zimbraPrefCalendarShowPastDueReminders = \u671f\u9650\u5207\u308c\u30ea\u30d
 MSG_zimbraPrefCalendarShowPastDueReminders = \u671f\u9650\u5207\u308c\u30ea\u30de\u30a4\u30f3\u30c0\u30fc\u3092\u6709\u52b9\u306b\u3059\u308b
 LBL_zimbraPrefAppleIcalDelegationEnabled = CalDAV\u30a4\u30f3\u30bf\u30fc\u30d5\u30a7\u30a4\u30b9\u7528\u306b\u5171\u6709\u30ab\u30ec\u30f3\u30c0\u30fc\u306eiCal\u59d4\u4efb\u30e2\u30c7\u30eb\u3092\u4f7f\u7528
 MSG_zimbraPrefAppleIcalDelegationEnabled = CalDAV\u30a4\u30f3\u30bf\u30fc\u30d5\u30a7\u30a4\u30b9\u7528\u306b\u5171\u6709\u30ab\u30ec\u30f3\u30c0\u30fc\u306eiCal\u59d4\u4efb\u30e2\u30c7\u30eb\u3092\u4f7f\u7528
+LBL_zimbraPrefMailDeliveryStatusNotification = \u914d\u9001\u30b9\u30c6\u30fc\u30bf\u30b9\u3092\u6709\u52b9\u306b\u3059\u308b
```

If "Enable delivery status notification" in English,
```suggestion
LBL_zimbraPrefMailDeliveryStatusNotification = \u914d\u9001\u72b6\u6cc1\u901a\u77e5\u3092\u6709\u52b9\u306b\u3059\u308b
```

If "Enable delivery status" in English,
```suggestion
LBL_zimbraPrefMailDeliveryStatusNotification = \u914d\u9001\u72b6\u6cc1\u3092\u6709\u52b9\u306b\u3059\u308b
```

---

Please check for replaced label.